packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -31,7 +30,7 @@ class ImageInfo {
   /// This is the object to pass to the [Canvas.drawImage],
   /// [Canvas.drawImageRect], or [Canvas.drawImageNine] methods when painting
   /// the image.
-  final ui.Image image;
+  final ui.Image? image;
```

Comment:
How can this become null? The constructor asserts that it is non-null.

Reply:
Ditto

Reply:
changed